targets/disc-configs/lenovo-x1-carbon-disk-config.nix

```diff
@@ -0,0 +1,166 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+_: {
+  disko.devices = {
+    disk.disk1 = {
+      device = "DRIVE_PATH";
+      type = "disk";
+      content = {
+        type = "gpt";
+        partitions = {
+          boot = {
+            name = "boot";
+            size = "1M";
+            type = "EF02";
+          };
```

Comment:
The EFI standard allows only one boot loader to be registered, how would the second partition used?

Reply:
Currently there is only one bootloader partition. The duplication in this case require some external tooling to switch between A and B, which we decided to avoid.